packages/react-components/src/templates/GroupProfileHeader.tsx

```diff
@@ -89,10 +90,15 @@ const GroupProfileHeader: React.FC<GroupProfileHeaderProps> = ({
         </div>
         <TabNav>
           <TabLink href={route.about({}).$}>About</TabLink>
-          <TabLink href={route.calendar({}).$}>Calendar</TabLink>
-          <TabLink href={route.upcoming({}).$ + queryParamString(searchQuery)}>
-            Upcoming Events
-          </TabLink>
+          {active && <TabLink href={route.calendar({}).$}>Calendar</TabLink>}
+          {active && (
```

Comment:
Very minor. What do you think about this?
```
    {active && (
      <TabLink href={route.calendar({}).$}>Calendar</TabLink>
      <TabLink href={route.upcoming({}).$ + queryParamString(searchQuery)}>
         Upcoming Events
       </TabLink>
     )}
```

Reply:
It would need to be within a fragment. I did similar originally but the tab nav inspects the first level of nodes and thus breaks when it finds a fragment. Doing the conditional statement twice seemed a pragmatic solution than trying to find tab links in a tree the other side.